pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java

```diff
@@ -185,7 +185,6 @@
     public boolean msgChunkPublished;
 
     private Optional<DispatchRateLimiter> dispatchRateLimiter = Optional.empty();
-    private Optional<SubscribeRateLimiter> subscribeRateLimiter = Optional.empty();
```

Comment:
Why move `subscribeRateLimiter` to the AbstractTopic?

Reply:
This can uniform the limiters to `AbstractTopic` and we can update it like [updateSubscribeRate](https://github.com/apache/pulsar/pull/14890/files?diff=split&w=0#diff-0210356c8a88e4efa89eb769a027fa6c166db479dbad8bbbbc704c6ed6e317f5R2337)

Reply:
But the `subscribeRateLimiter` seems only work on persistent topics. Could we only make it update on persistent topics?

Reply:
OK, PTAL :) @RobertIndie